fuel-tx/src/receipt.rs

```diff
@@ -156,6 +156,51 @@ pub enum Receipt {
     },
 }
 
+impl core::fmt::Display for Receipt {
```

Comment:
Why can't we derive the implementation with all fields?=D

Reply:
Regarding all fields - I thought it may be just a bit too much details, which, when needed, could be retrieved by `Debug`. I prefer `Display` to be more concise, targeted on humans, not developers :)

I have no strong opinions here, so if you prefer to have all fields, I can just stick `derive_more::Display` without any custom `#[display(fmt =` attributes.

Let me know what you think.